internal/benchmarks/benchmarks_test.go

```diff
@@ -154,6 +157,38 @@ func BenchmarkWriteAmplification(b *testing.B) {
 	})
 }
 
+func BenchmarkReadSchema(b *testing.B) {
+	ctx := context.Background()
+	testSchema := testutils.TestSchema()
+	var opts []roll.Option
+
+	testutils.WithMigratorInSchemaAndConnectionToContainerWithOptions(b, testSchema, opts, func(mig *roll.Roll, db *sql.DB) {
+		b.Cleanup(func() {
+			require.NoError(b, mig.Close())
+		})
+
+		setupInitialTable(b, ctx, testSchema, mig, db, 1)
```

Comment:
it seems clear that for the shake of this test and the others, we probably want a more complex schema here, right? not for this PR

Reply:
Yeah, agree. I'm not sure exactly how the performance of `read_schema` would change depending on the complexity of the schema but I guess that's something we could actually benchmark too :)

Reply:
++